Content.Server/_DV/RemoteControl/EntitySystems/RemoteControlSystem.cs

```diff
@@ -0,0 +1,216 @@
+    /// a single unit.
+    /// </summary>
+    /// <param name="holder">Entity holding the remote control.</param>
+    /// <param name="recipient">Optional recipient, if set then only that specific entity will receive the order.</param>
+    /// <returns>True if the order was sent successfuly, false otherwise.</returns>
+    public bool SendSelfPointOrder(Entity<RemoteControlHolderComponent?> holder, EntityUid? recipient = null)
+    {
+        if (!Resolve(holder, ref holder.Comp))
+            return false;
+
+        if (!TryComp<RemoteControlComponent>(holder.Comp.Control, out var controlComp))
+            return false;
+
+        var control = (holder.Comp.Control, controlComp);
+        if (!CanSendOrder(control))
+            return false;
+
+        var ev = new RemoteControlSelfPointOrderEvent(holder, holder.Comp.Control, controlComp.BoundEntities);
+        if (recipient.HasValue)
+        {
+            RaiseLocalEvent(recipient.Value, ref ev);
+            _useDelay.TryResetDelay(holder.Comp.Control);
+        }
+        else
+            SendOrderToReceivers(control, ref ev);
+
+        return true;
+    }
+
+    /// <summary>
+    /// Sends an order freeing the bound entities on the same channel, or optionally to just
+    /// a single unit.
+    /// </summary>
+    /// <param name="holder">Entity holding the remote control.</param>
+    /// <param name="recipient">Optional recipient, if set then only that specific entity will receive the order.</param>
+    /// <returns>True if the order was sent successfuly, false otherwise.</returns>
+    public bool SendFreeUnitOrder(Entity<RemoteControlHolderComponent?> holder, EntityUid? recipient = null)
+    {
+        if (!Resolve(holder, ref holder.Comp))
+            return false;
+
+        if (!TryComp<RemoteControlComponent>(holder.Comp.Control, out var controlComp))
+            return false;
+
+        var control = (holder.Comp.Control, controlComp);
+        if (!CanSendOrder(control))
+            return false;
+
+        var ev = new RemoteControlFreeUnitOrderEvent(holder, holder.Comp.Control, controlComp.BoundEntities);
+        if (recipient.HasValue)
+        {
+            RaiseLocalEvent(recipient.Value, ref ev);
+            _useDelay.TryResetDelay(holder.Comp.Control);
+        }
+        else
+            SendOrderToReceivers(control, ref ev);
+
+        return true;
+    }
+
+    /// <summary>
+    /// Handles when an entity using an active remote control points at an entity.
+    /// </summary>
+    /// <param name="holder">Entity holding/wearing the remote control.</param>
+    /// <param name="args">Args for the event, notably the entity pointed at.</param>
+    private void OnPointedAtEntity(Entity<RemoteControlHolderComponent> holder, ref AfterPointedAtEvent args)
+    {
+        if (holder.Owner == args.Pointed)
+            SendSelfPointOrder(holder.AsNullable());
+        else
+            SendEntityPointOrder(holder.AsNullable(), args.Pointed);
```

Comment:
May be a good spot for the free order?

```suggestion
        else if (
          TryComp<RemoteControlReceiverComponent>(args.Pointed, out var receiver)
          && receiver.BoundController == holder.Comp.Control
        )
            SendFreeUnitOrder(holder.AsNullable());
        else
            SendEntityPointOrder(holder.AsNullable(), args.Pointed);
```

Reply:
Sure, I think it's a decent way of giving freedom back to the unit.